test/integration/error-node-builtin-unresolved/expected-output.txt

```diff
@@ -1,6 +1,4 @@
 [snowpack] ! installing dependencies…
 [snowpack] ../../../node_modules/bad-node-builtin-pkg/entrypoint.js
-   "XXXX" (Node.js built-in) could not be resolved. (https://www.snowpack.dev/#node-built-in-could-not-be-resolved)
-[snowpack] "XXXX" (Node.js built-in) could not be resolved. (https://www.snowpack.dev/#node-built-in-could-not-be-resolved). See https://www.snowpack.dev/#troubleshooting
-[snowpack] Entry module cannot be external (http).
-[snowpack] install skipped (nothing to install)
\ No newline at end of file
+   Import "http" (Node.js built-in) is not available in the browser. Run with --polyfill-node to fix.
+[snowpack] Install failed.
```

Comment:
nit: sometimes we start with a capital letter (`installing dependencies…`), sometimes we don’t (`Install failed.`). I’d be on board with making it consistent, whatever that means.

Reply:
+1, I noticed this as well

Reply:
+1, noticed this as well but wasn't sure how to resolve in this PR